api/data_explorer/controllers/export_url_controller.py

```diff
@@ -236,20 +236,18 @@ def _get_range_clause(column, value):
 def _get_facet_and_value(filter):
     facets = current_app.config['UI_FACETS']
     split = filter.rsplit('=', 1)
```

Comment:
nit: I know this wasn't added, but `filter` is also a built-in python function https://docs.python.org/2/library/functions.html#filter. Maybe `filter_value`?